backend/pyproject.toml

```diff
@@ -91,6 +91,10 @@ backend.help = "Runs the Unstract backend"
 migrate_db.cmd = "python manage.py migrate"
 migrate_db.env_file = ".env"
 migrate_db.help = "Performs DB migrations for Unstract backend"
+# Commands for backend db migration
+create_schema.cmd = "python manage.py create_v2_schema"
+create_schema.env_file = ".env"
+create_schema.help = "Create schema"
```

Comment:
```suggestion
```
@muhammad-ali-e @hari-kuriakose this feels like a command we'd need to run once only. I don't think this will be useful after the migration so its not a useful command to add in this file.

Maybe if the command is generic - `create_schema` this could have made sense but we removed / avoided adding `SCHEMAS_TO_MIGRATE` env in the backend env for a similar reason

Reply:
I disagree with the last point. We removed `SCHEMAS_TO_MIGRATE` because it wasn't consistently useful for `OSS users`. However, the PDM commands here are primarily meant for `developers`, not `end-users`.

That said, I do agree with your first point—we’d likely only need to run this command once during migration. Given that, we can consider removing it from this file.
cc: @hari-kuriakose for your opinion